project/credentials.sbt

```diff
@@ -0,0 +1,4 @@
+credentials += Credentials("Sonatype Nexus Repository Manager",
+        "oss.sonatype.org",
+        sys.env.get("MAVEN_LOGIN").getOrElse("NO_MAVEN_LOGIN_SPECIFIED"),
```

Comment:
I'm wondering if it would be more direct to throw here if the envvar is not specified. Why do you try to login as NO_MAVEN_LOGIN_SPECIFIED?

Reply:
Sorry, this file is not needed. I accidentally added it, I need to remove it.